src/containers/InfoProyecto/index.js

```diff
@@ -144,107 +136,117 @@ export default function InfoProyecto({ projectData, type, state }) {
+                  </Typography>
+                </Box>
+              )}
+            </List>
           </Box>
           <Divider style={{ marginBottom: 15, marginTop: 15 }} />
 
           <Box>
-            <Accordion>
-              <AccordionSummary
-                expandIcon={<ExpandMoreIcon />}
-                aria-controls="panel1a-content"
-                id="panel1a-header"
-              >
-                <Typography style={{ fontWeight: 600 }} variant="body1">
-                  Personas Asignadas
-                </Typography>
-              </AccordionSummary>
-              <AccordionDetails>
-                <List
-                  sx={{
-                    width: "100%",
-                    maxWidth: 360,
-                    // bgcolor: "rgb(240,240,240)",
-                    position: "relative",
-                    overflow: "auto",
-                    maxHeight: 370,
-                    "& ul": { padding: 0 },
-                  }}
-                  subheader={<li />}
-                >
-                  {projectData.people.length != 0 ? (
-                    projectData.people.map((person) => {
-                      return (
-                        <>
-                          <ListItem key={person.id} role="listitem">
-                            <ListItemText primary={person.full_name} />
-                          </ListItem>
-                        </>
-                      );
-                    })
-                  ) : (
-                    <Box style={{ paddingTop: 20 }}>
-                      <Typography variant="button" display="block" gutterBottom>
-                        Aún no hay personas asociadas
-                      </Typography>
-                    </Box>
-                  )}
-                </List>
-              </AccordionDetails>
-            </Accordion>
+            <Typography style={{ fontWeight: 600 }} variant="body1">
+              Personas Asignadas
+            </Typography>
+            <List
+              sx={{
+                width: "100%",
+                maxWidth: 360,
+                position: "relative",
+                overflow: "auto",
+                maxHeight: 370,
+                "& ul": { padding: 0 },
+              }}
+              subheader={<li />}
+            >
+              {projectData.people.length != 0 ? (
+                projectData.people.map((person) => {
+                  return (
+                    <>
+                      <ListItem key={person.id} role="listitem">
+                        <ListItemText primary={person.full_name} />
+                      </ListItem>
+                    </>
+                  );
+                })
+              ) : (
+                <Box style={{ paddingTop: 20 }}>
+                  <Typography variant="button" display="block" gutterBottom>
+                    Aún no hay personas asociadas
+                  </Typography>
+                </Box>
+              )}
+            </List>
```

Comment:
I think this could be a separate component. Also, it's duplicated with the "Tecnologias" block.